app/assets/stylesheets/new_board/_story.scss

```diff
@@ -0,0 +1,130 @@
+.Story {
+  display: flex;
+  justify-content: space-between;
+  padding: 7px 8px;
+  background-color: $story-background-color;
+  border-bottom: 1px solid $story-border-bottom-color;
+  border-top: 1px solid $story-border-top-color;
+  min-height: 37px;
+  cursor: move;
+  font-size: 12px;
+
+  &:hover {
+    background-color: $story-hover-background-color;
+  }
+
+  &--unestimated {
+    font-style: italic;
+  }
+
+  &--release {
+    color: $white;
+    background-color: $blue-4;
+    border-top-color: $blue-5;
+    border-bottom-color: $blue-3;
+
+    a {
+      color: $white;
+    }
+
+    &:hover {
+      background-color: $blue-3;
+    }
+  }
+
+  &__info {
+    flex: 1;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-left: 15px;
+  }
+
+  &__icon {
+    display: flex;
+    justify-content: flex-start;
+    align-items: center;
+    margin-right: 5px;
+  }
+
+  &__title {
+    flex: 1;
+  }
+
+  &__actions {
+    display: flex;
+    align-items: center;
+  }
+
+  &__estimated {
+    display: inline-flex;
+    align-items: center;
+    justify-content: center;
+    background: $estimated-background-color;
+    margin-right: 1px;
+    padding: 1px 1px;
+    border: 1px solid $estimated-border-color;
+    border-radius: 3px;
+    line-height: 1;
+    min-width: 15px;
+    font-style: normal;
+  }
+
+  &__estimate {
+    display: inline-flex;
+    align-items: center;
+    justify-content: center;
+    margin-right: 1px;
+    padding: 1px 1px;
+    line-height: 1;
+    font-weight: bold;
+    color: $estimate-background-color;
+    cursor: pointer;
+    font-style: normal;
+  }
+
+  &__label {
+    padding-right: 3px;
+    color: $story-label-color;
+    text-decoration: none;
+  }
+
+  &__labels {
+    color: $story-labels-color;
+    font-size: 84%;
```

Comment:
It looks like a magic number. Are you sure about it? I think there are limited approaches to use percentage in font-size.

Reply:
But, this font-size there's in actual Central, I just copy. What do you thing better? keep it or change it?

Reply:
But, this font-size there's in actual Central, I just copy. What do you thing better? keep it or change it?

Reply:
I'm not sure if I understood correctly. Did you copy it from somewhere or it's already been here?

Reply:
We only migrate the css to our new board, so in there, the actual central to specify the size's font, it put 84% on it. Can you have another idea to make this part better?!

Reply:
Yeah. I have two suggestionsyou need to make sure that percentage isn't necessary, changing for px, em or rem, or you could put this value in a variable to give it a meaning.